CHANGELOG.md

```diff
@@ -185,6 +185,8 @@ This was introduced in [pull request #2844: Remove compatibility mode from govuk
 
 ### Fixes
 
+In [pull request 2851: Support Prototype Kit v13](https://github.com/alphagov/govuk-frontend/pull/2851) we introduced support for the plugins system introduced in the upcoming Prototype Kit v13.
```

Comment:
```suggestion
In [pull request 2851: Support Prototype Kit v13](https://github.com/alphagov/govuk-frontend/pull/2851) we introduced support for the plugins system included in the upcoming Prototype Kit v13.
```

Reply:
Nice spot, double "introduced"! Pushed it up

@36degrees @claireashworth Good to go?